substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/ClassEntry.java

```diff
@@ -90,12 +90,12 @@ public class ClassEntry extends StructureTypeEntry {
      * A list of all files referenced from info associated with this class, including info detailing
      * inline method ranges.
      */
-    private ArrayList<FileEntry> files;
+    final private ArrayList<FileEntry> files;
```

Comment:
Please use `private final` here and below to make checkstyle happy

Reply:
Oops, sorry about that. Done.